### PR TITLE
[Example] Remove Bias in Linear layer because of BatchNorm1d usage

### PR DESCRIPTION
**Pull request by decoherencer**
## Description
The GIN MLP has linear layers which don't need bias as the BatchNorm1d is applied just after the MLP at [here](https://github.com/dmlc/dgl/blob/9922f41f06e53aa5eb75a68bea615ed59d2e0b03/examples/pytorch/gin/gin.py#L79).
Hence, removing the bias [could](https://twitter.com/karpathy/status/1013245864570073090) speed up the learning.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by mufeili — APPROVED** [review]

**Comment by mufeili** [comment]
Thanks.
